src/tests/multihost/ipa/test_adtrust.py

```diff
@@ -116,6 +116,45 @@ def test_pam_sss_gss_handle_large_krb_ticket(self, multihost,
         cmd = f'mv {bk_krbkcm} {krbkcm}'
         multihost.client[0].run_command(cmd, raiseonerr=False)
 
+    def test_adgrpwith_at_ratesign(self, multihost):
+        """
+        :title: user membership of AD group with @ sign
+        :id: ee9ca809-6ea7-48f2-a0fe-d9eccadf5d81
+        :customerscenario: True
+        :bugzilla:
+         https://bugzilla.redhat.com/show_bug.cgi?id=2061795
+        :description:
+         In the IPA-AD trust, IPA-user having group membership of an
+         AD-group, containing '@' sign in it's name, should be resolvable
+         with default re_expression on a ipa-client system.
+        :steps:
+          1. Create an AD-group having '@' sign in it's name
+          2. Create an AD-user and add it to above created group
+          3. From ipaclient, confirm ad-user is showing correct group
```

Comment:
Steps 3. and 4. seems to be in reverse order in the test.

Reply:
Corrected the order